Lib/fontTools/subset/cff.py

```diff
@@ -401,9 +418,24 @@ def op_callgsubr(self, index):
 		psCharStrings.SimpleT2Decompiler.op_callgsubr(self, index)
 		self.processSubr(index, subr)
 
+	def stop_hint_count(self, index=None):
```

Comment:
I think you can remove `index` from the parameter list as it's not used in this method, nor in the calls to this method.

Reply:
Actually, it is used. All of the functions in the stop_hintcount_ops list are called with two arguments: self , and index. stop_hint_count() needs accept the second argument, even though it isn't used in this functions. These functions are all called by  misc.psCharstrings.SimpleT2Decompiler.execute(), with the same handler call.

Reply:
OK.

Reply:
if it's unused, it's clearer if you just use `*args, **kwargs` then

Reply:
Agreed.